p22/benches/bench.rs

```diff
@@ -0,0 +1,163 @@
+#![feature(test)]
+extern crate test;
+
+#[cfg(test)]
+mod tests {
+    use p22::calc::*;
+    use p22::figures::*;
+    use p22::tictac::*;
+    use std::hint::black_box;
+    use test::Bencher;
+
+    #[bench]
+    fn benchmark_celsius2farenheit(b: &mut Bencher) {
+        b.iter(|| black_box(celsius2farenheit(10)));
+    }
```

Comment:
You needed to measure performance only of the Fibonacci functions to see that the loop function is much faster than the recursive one.